src/pages/en/tutorial/3-components/1.md

```diff
@@ -87,18 +89,70 @@ This allows you to get started quickly with any working code, often duplicated t
 
 ### Test your knowledge
 
-Use these words in the text below. Click the blanks to reveal if you're right!
 
-<Blanks>
-- reusable
-- `src/pages`
-- refactor
-- standard HTML elements
-</Blanks>
-
-`.astro` files can use || **standard HTML elements** || and other Astro components in the same template, alongside each other. This means that you can decide when and where to || **refactor** || your code, creating || **reusable** || fragments of HTML defined in an Astro component. 
-
-These components should live in a folder outside of || **`src/pages`** || since they are not meant to be pages on your website.
+1. Astro files can use these alongside Astro components in their template:
+  <MultipleChoice>
+    <Option>
+      reusable
+    </Option>
+    <Option>
+      `src/pages/`
+    </Option>
+    <Option>
+       refactor
+    </Option>
+    <Option isCorrect>
+       standard HTML elements
+    </Option>
+  </MultipleChoice>
+
+2. You can do this when you have elements repeated on multiple pages:
+  <MultipleChoice>
+      <Option>
+        reusable
+      </Option>
+      <Option>
+        `src/pages/`
+      </Option>
+      <Option isCorrect>
+        refactor
+      </Option>
+      <Option>
+        standard HTML elements
+      </Option>
+    </MultipleChoice>
+
+3. Astro components give you fragments of HTML that are:
+  <MultipleChoice>
+    <Option isCorrect>
+      reusable
+    </Option>
+    <Option>
+      `src/pages/`
+    </Option>
+    <Option>
+       refactor
+    </Option>
+    <Option>
+       standard HTML elements
+    </Option>
+  </MultipleChoice>
+
+4. Unless you want to make a new page on your site, don't put your components here:
+  <MultipleChoice>
+    <Option>
+      reusable
+    </Option>
+    <Option isCorrect>
+      `src/pages/`
+    </Option>
+    <Option>
+       refactor
+    </Option>
+    <Option>
+       standard HTML elements
+    </Option>
+  </MultipleChoice>
```

Comment:
This one feels like it might benefit from more similar options to force the reader to think a bit more — not sure how you feel about that.

```suggestion
4. Unless you want to make a new page on your site, don't put your components here:
  <MultipleChoice>
    <Option>
      `src/layouts/`
      <Fragment slot="feedback">
         Try again! A `layouts` directory is a good place to put any layout components you build and won’t create new pages on your site.
       </Fragment>
    </Option>
    <Option isCorrect>
      `src/pages/`
    </Option>
    <Option>
       `src/components/`
       <Fragment slot="feedback">
         Try again! A `components` directory is a good place for your components to live and won’t create new pages on your site.
       </Fragment>
    </Option>
  </MultipleChoice>
```

Reply:
I also like this, but we haven't done layouts yet! So, I'll work on this in the same vein.